aws/rds.go

```diff
@@ -0,0 +1,95 @@
+package aws
+
+import (
+	"time"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+)
+
+func waitUntilRdsDeleted(svc *rds.RDS, input *rds.DescribeDBInstancesInput) error {
```

Comment:
Do you think it is possible to use [WaitUntilDBInstanceDeleted](https://docs.aws.amazon.com/sdk-for-go/api/service/rds/#RDS.WaitUntilDBInstanceDeleted) instead? I know the AWS wait functions can be finicky, but it might work here.

Note: not worth it for the v1, but probably should capture as a comment to indicate the possibility!

Reply:
It definitly does, maybe using `WaitUntilDBInstanceDeletedWithContext` that I can set timeouts.